source/dub/commandline.d

```diff
@@ -257,6 +257,7 @@ struct CommonOptions {
 	/// Parses all common options and stores the result in the struct instance.
 	void prepare(CommandArgs args)
 	{
+		registry_urls = environment.get("DUB_REGISTRY", null).split(";");
```

Comment:
We could add env var support to `CommandArgs` if something more structured seems appropriate.